packages/editor/src/client/panel/embedEditPanel.tsx

```diff
@@ -114,29 +117,33 @@ export function EmbedEditPanel({value, onClose, onConfirm}: EmbedEditPanel) {
       <PanelSection>
         <TextArea
           marginBottom={Spacing.ExtraSmall}
-          label="Embed"
+          label={t('Embed')}
           errorMessage={errorMessage}
           value={input}
           onChange={e => setInput(e.target.value)}
         />
         <Box marginBottom={Spacing.ExtraSmall}>
           <Typography variant="subtitle1" spacing="small">
-            Facebook Post/Video, Instagram Post, Twitter Tweet, Vimeo Video, YouTube Video formatted
-            like e.g.
+            {t(
+              'Facebook Post/Video, Instagram Post, Twitter Tweet, Vimeo Video, YouTube Video formatted like e.g.'
+            )}
           </Typography>
           <code>https://www.facebook.com/id/posts/id/</code>
           <Typography variant="subtitle1" spacing="small">
-            Embed codes attributes <code>title, src, width, height</code> are validated e.g.
+            {t('Embed codes attributes <code>title, src, width, height</code> are validated e.g.')}
```

Comment:
It's not really useful to use such a long key.

Reply:
Done